calico_cni/policy_drivers.py

```diff
@@ -320,6 +307,25 @@ def _escape_chars(self, unescaped_string):
         return re.sub('[^a-zA-Z0-9\.\_\-]', swap_char, unescaped_string)
 
 
+class DefaultDenyInboundDriver(KubernetesAnnotationDriver):
+    """
+    This driver sets the labels and correct profile on the endpoint.
+    """
+    def apply_profile(self, endpoint):
+        profile_id = "k8s_ns.%s" % self.namespace
+        _log.debug("Constructed profile ID - %s" % profile_id)
+        pod = self._get_api_pod()
+        labels = pod["metadata"].get("labels", {})
+        labels["calico/k8s_ns"] = self.namespace
+        _log.debug("Got labels - %s" % labels)
+        endpoint.labels = labels
+        endpoint.profile_ids = [profile_id]
+        self._client.update_endpoint(endpoint)
+
+    def remove_profile(self):
+        _log.debug("Nothing to do")
```

Comment:
More friendly/useful log plz (I understand that this is a "get it to casey fast" iteration for now)